.eslintrc.json

```diff
@@ -28,12 +30,34 @@
     "ecmaVersion": 2019,
     "sourceType": "module"
   },
+  "settings": {
+    "import/resolver": {
+      "typescript": {}
+    }
+  },
   "rules": {
     "no-console": "warn",
     "no-debugger": "error",
-    "import/order": "warn",
+    "sort-imports": "off",
+    "import/order": "off",
     "import/export": "off",
+    "import/first": "error",
+    "import/newline-after-import": "error",
     "import/no-extraneous-dependencies": "error",
+    "import/no-duplicates": "error",
+    "simple-import-sort/imports": [
+      "error",
+      {
+        "groups": [
+          ["^\\u0000", "/(fixtures|mocks|patches)$"],
+          ["^@?\\w"],
```

Comment:
Isn't this mixing imports of installed packages and import via the `@` alias which is in fact an absolute path? I would rather split this and make the alias imports after all packages, as it is 1st party.

Reply:
No, because this matches `@` only if it's followed by a `\w`, so interpreting this as a scoped external package. The `@` alias actually ends up in the `^` match, grouped right after any of our mono-repo imports, if any.